scipy/stats/_distr_params.py

```diff
@@ -111,6 +111,7 @@
 
 distdiscrete = [
     ['bernoulli',(0.3,)],
+    ['betabinom', (5, 2.3098496451481823, 0.62687954300963677)],
```

Comment:
Any specific reason to not use e.g. 2.3 and 0.63 ?

Reply:
None at all. I copy-pasted that from another distribution.

Reply:
Fixed.